### PR TITLE
Added support for Telegram Stickers and New Saving Mechanism

### PR DESCRIPTION
**Pull request by BlueCatSoftware**

### CODE REVIEW

**Comment by PranavPurwar** [comment]
ong

File: app/src/main/java/com/emojimixer/activities/MainActivity.java [hunk 1]
```diff
@@ -125,17 +130,34 @@ public void initLogic() {
...
                 } else {
                     ActivityCompat.requestPermissions(MainActivity.this, new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE}, 1);
                 }
             }
 
         });
 
+        exportEmoji.setOnClickListener(v ->{
+            BottomSheetDialog sheetDialog = new BottomSheetDialog(this);
+            View view = getLayoutInflater().inflate(R.layout.layout, null);
```
Comment (ilyassesalama) [inline app/src/main/java/com/emojimixer/activities/MainActivity.java:146]: Use more descriptive IDs.

File: app/src/main/java/com/emojimixer/adapters/EmojisSliderAdapter.java [hunk 2]
```diff
@@ -77,7 +78,6 @@ private void loadEmojiFromUrl(ImageView image, CircularProgressIndicator progres
         Glide.with(mContext)
                 .load(url)
                 .fitCenter()
-                .transition(DrawableTransitionOptions.withCrossFade())
```
Comment (ilyassesalama) [inline app/src/main/java/com/emojimixer/adapters/EmojisSliderAdapter.java:80]: Why did you remove the cross-fade transition?

File: app/src/main/java/com/emojimixer/functions/Utils.java [hunk 3]
```diff
@@ -37,5 +62,80 @@ public static int getRecyclerCurrentItem(RecyclerView recyclerView, SnapHelper s
         }
     }
 
+    public static void saveImage(ImageView imageView, Context context, String unicodeRep, boolean exportTelegram) {
```
Comment (ilyassesalama) [inline app/src/main/java/com/emojimixer/functions/Utils.java:65]: Instead of digging into more mess, you could've saved the emoji directly from its URL.

**Review by ilyassesalama — CHANGES_REQUESTED** [review]
Can you change what has been mentioned and commit them again?
